### DISCUSSION TITLE
360° Video Keys for rotate and up and down

### DISCUSSION THREAD

**Discussion by DeltaTango69** [immich-app/immich#13123] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I  like to watch our 360° video footage on the big TV. For steering in the video we use a small remote mouse with touchpad and keypad. But it is a bit tricky to find the small buttons for steering the video:

![image](https://github.com/user-attachments/assets/8e382bb8-c64c-481b-a091-4f5fc18a9d12)

Is there a chance to use for exampe also the (WASD) Keys for that and for zooming maybe  (arrow up and down)?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by orrd** [comment]
I didn't think it could display 360 video (but it does photos).  Are you able to view 360 videos with Immich?

**Reply by DeltaTango69** [reply]
Yes of course. But only in the browser. Otherwise there were no reason for the request.

**Reply by orrd** [reply]
What type of camera and video format is it?  It doesn't work with the videos from my Theta camera, it just shows the video in 2D mode (showing two circles).  And I've seen posts saying it doesn't work with other 360 format videos.  Edit:  Actually Theta videos do work if converted first using the Theta app.  Good to know.
